Web UI/CognitiveSearch.UI/appsettings.json

```diff
@@ -5,37 +5,20 @@
       "Default": "Warning"
     }
   },
-
-    // Required fields
```

Comment:
Please rotate the keys, and I think there is a way in git to get rid of a specific commit from the logs.

Reply:
Thanks, ran git revert and re-added back the changes.